compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUVectorization.cpp

```diff
@@ -30,7 +30,8 @@ static void populateVectorizationPatterns(RewritePatternSet &patterns) {
   MLIRContext *ctx = patterns.getContext();
   linalg::LinalgVectorizationOptions opt;
   linalg::LinalgTransformationFilter f(
-      StringAttr::get(ctx, getVectorizeMarker()));
+      {StringAttr::get(ctx, getWorkgroupKTiledMarker()),
+       StringAttr::get(ctx, getVectorizeMarker())});
```

Comment:
This broke our gcc build:

```
[2022-08-29T18:32:31Z] /usr/src/github/iree/compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUVectorization.cpp:34:51: error: call of overloaded 'LinalgTransformationFilter(<brace-enclosed initializer list>)' is ambiguous
[2022-08-29T18:32:31Z]    34 |        StringAttr::get(ctx, getVectorizeMarker())});
[2022-08-29T18:32:31Z]       |                                                   ^
```

https://buildkite.com/iree/iree-build-configurations/builds/4369#0182eac9-c6a8-409a-ae88-c93c22d9fe0b

Reply:
I think this should fix it:
https://github.com/iree-org/iree/pull/10235

Reply:
Thanks, confirmed fixed now: https://buildkite.com/iree/iree-build-configurations/builds/4373

Reply:
Thanks, Scott